composer.json

```diff
@@ -11,10 +11,10 @@
         "chat": "https://www.hipchat.com/gIdidQscL"
     },
     "require": {
-        "php":           ">=5.6.0",
-        "ext-openssl":    "*",
-        "guzzlehttp/guzzle": "~6.0",
-        "guzzlehttp/guzzle-services": "~1.0",
+        "php": ">=7.0",
+        "ext-openssl": "*",
+        "guzzlehttp/guzzle": "~7.0",
```

Comment:
I'd rather keep `^6.0 || ^7.0` (and 1.0 as min version for `guzzlehttp/guzzle-services`, to stay compatible with projects which are still using Guzzle 6.

Reply:
also not changing the min PHP version then

Reply:
I updated the versions constraints to support legacy projects